app/javascript/pages/admin/Events/edit.js

```diff
@@ -8,21 +8,50 @@ import { graphQLError } from 'actions'
 
 import EventForm from './form'
 import Loading from 'components/LoadingIcon'
+import DropDownMenu from 'material-ui/DropDownMenu'
+import MenuItem from 'material-ui/MenuItem'
 
 import EventQuery from './queries/show.gql'
 import UpdateEventMutation from './mutations/update.gql'
+import NamedAvatar from 'components/NamedAvatar'
+import DestroySignupMutation from 'mutations/DestroySignupMutation.gql'
+import CreateSignupMutation from 'mutations/CreateSignupMutation.gql'
 
-const EditEvent = ({ data: { networkStatus, event, eventTypes, offices, organizations }, updateEvent }) =>
+const EditEvent = ({
+  data: { networkStatus, event, eventTypes, offices, organizations, users },
+  updateEvent,
+  destroySignup,
+  createSignup,
+}) =>
   networkStatus === NetworkStatus.loading ? (
     <Loading />
   ) : (
-    <EventForm
-      event={event}
-      eventTypes={eventTypes}
-      offices={offices}
-      organizations={organizations}
-      onSubmit={updateEvent}
-    />
+    <div>
+      <EventForm
+        event={event}
+        eventTypes={eventTypes}
+        offices={offices}
+        organizations={organizations}
+        onSubmit={updateEvent}
+      />
+      Signed up Users
+      {event.users.map((user, i) => (
+        <div key={i}>
```

Comment:
can we maybe extract and reuse the user list that is on the regular event details page? Then we could just pass in a prop if someone is an admin to have the add/remove controls added.